.gitignore

```diff
@@ -0,0 +1,4 @@
+*.egg-info/
+*.pyc
+.eggs/
```

Comment:
I suspect you meant

~~~~~~ text
/.eggs
/.tox
~~~~~~

since these are only expected at the top level.  See the last item in the first list in [Pattern Format](https://git-scm.com/docs/gitignore#_pattern_format).

Reply:
(For some reason I also always use the `.tox/` spelling -- even if it is less technically correct, I find it more aesthetically pleasing, I guess?  And it's not like we'd ever want to git add a .tox if it somehow showed up deeper in the tree.)

Reply:
It's easy to do; I often have to make a correction if I start a new **.gitignore** from scratch.

Generally, I'd rather something point out that I had a surprising item pop up than just ignore it; not only do I not want to **git add** it, I want something to tell me about such a surprise.  Having git point it out to me as untracked seems to make sense, 'cause I let git point out just that kind of thing about other files/directories that I might have forgotten to **git add**.

Reply:
I actually like the fact, `.tox/` indicates a folder but I could also go with `/.tox/`.